007_team_5_agent_design/007_team_5_agent_design.tex

```diff
@@ -1 +1,522 @@
+
+\subsection*{Heterogeneous Tower Experiments}
+These experiments compare the Team 5 agent with the agents designed by the other agent teams. The base parameters for all simulations are as follows:
+\begin{itemize}
+    \item 10 Team 5 agents, and two agents from each other team, excluding the random and selfish agents (i.e. half of the tower is Team 5).
+    \item 6 food per agent
+    \item 3 day reshuffle period
+    \item 20 day simulation
+\end{itemize}
+
+\subsubsection*{Simulation Length}
+\begin{figure}
+    \centering
+    \begin{minipage}{0.45\textwidth}
+        \begin{tikzpicture}
+            \begin{axis}[
+                width=\textwidth,
+                ymin=0,
+                xmin=0,
+                xlabel=Length of Simulation,
+                ylabel=Age,
+                ymajorgrids=true,
+                xmajorgrids=true,
+                grid style=dashed,
+            ]
+            \addplot[mark=*,blue]
+                coordinates{
+                    (20, 0)
+                    (30, 0)
+                    (40, 18)
+                    (50, 18.75)
+                    (60, 25)
+                    (70, 22)
+                    (80, 40)
+                    (90, 31)
+                    (100, 33)
+                };
+            \end{axis}
+            \end{tikzpicture}
+            \caption{Average age upon death vs length of simulation. Points with age 0 mean no agent died during those simulations.}
+            \label{fig:team5-exp-sim-length}
+    \end{minipage}
+    \hspace{0.05\textwidth}
+    \begin{minipage}{0.45\textwidth}
+        \begin{tikzpicture}
+            \begin{axis}[
+                width=\textwidth,
+                ymin=0,
+                xmin=0,
+                xlabel=Food per agent,
+                ylabel=Deaths,
+                ymajorgrids=true,
+                xmajorgrids=true,
+                grid style=dashed,
+            ]
+            \addplot[mark=*,blue]
+                coordinates{
+                    (3, 7)
+                    (4, 6)
+                    (5, 2)
+                    (6, 0)
+                    (7, 1)
+                };
+            \end{axis}
+            \end{tikzpicture}
+            \caption{Number of deaths vs food per agent ratio}
+            \label{fig:team5-exp-food-per-agent}
+    \end{minipage}
+\end{figure}
+\ToDo{Check this result: is it actually true?}
+Figure \ref{fig:team5-exp-sim-length} shows the relationship between the length of the simulation and the average age of Team 5 agents when they die: the graph shows a positive trend of increasing average age, implying that Team 5 agents perform better over time.
```

Comment:
I think these simulations could do with re-running if we can? It looks like there's an anomoly at 80 days which would probably go away with some more averaging. This would hopefully make the positive trend more clear